app/services/db-export/send-to-s3-bucket.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+/**
+ * Sends a file to our AWS S3 bucket
+ * @module SendToS3BucketService
+ */
+
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const S3Config = require('../../../config/s3.config')
+const path = require('path')
+
+/**
+ * Sends a file to our AWS S3 Bucket using the filePath that it receives
+ *
+ * @param {String} filePath A string containing the path of the file to send to the S3 bucket
+ *
+ * @returns {Boolean} True if the file is uploaded successfully and false if not
+ */
+async function go (filePath) {
+  if (!filePath) {
+    global.GlobalNotifier.omfg('ERROR uploading file to S3 bucket. No file path given')
+    return false
+  }
```

Comment:
Typically a service like this won't be used in isolation. It will be called by a marshalling service or function, that is taking the result of one thing and passing it to this.

When this is the case we let that 'thing' handle situations where something is unexpected. It's fine to code the `SendToS3BucketService` with the assumption that it will always be given a path, and should it not or that path be invalid then things will go 💥 !

```suggestion
```